capella2polarion/converters/converter_config.py

```diff
@@ -473,3 +466,25 @@ def merge_converters(
             result[key] = value
 
     return result
+
+
+def _get_valid_converters() -> set[str]:
```

Comment:
It should be safe to assume that this set, once computed, will no longer change. So we could memoize the function and freeze the returned set to reduce a little bit of overhead:

````python
import functools

@functools.cache
def _get_valid_converters() -> frozenset[set]:
    ...
    return frozenset(valid_converters)
````